table/table.go

```diff
@@ -210,9 +210,9 @@ func (t *Table) Height(h int) *Table {
 	return t
 }
 
-// Offset sets the table rendering offset.
-func (t *Table) Offset(o int) *Table {
-	t.offset = o
+// YOffset sets the table rendering offset.
+func (t *Table) YOffset(o int) *Table {
+	t.yOffset = o
 	return t
 }
```

Comment:
In theory you could keep the original `Offset` function and add a comment marking it as deprecated:

https://go.dev/wiki/Deprecated

No strong opinion, just commenting about a possibility.

Reply:
Sure, it's fine to remove it IMO